package/skelcd-control-MicroOS.spec

```diff
@@ -12,50 +12,34 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
-######################################################################
-#
-# IMPORTANT: Please do not change the control file or this spec file
-#   in build service directly, use
-#   https://github.com/yast/skelcd-control-MicroOS repository
-#
-#   See https://github.com/yast/.github/blob/master/CONTRIBUTING.md
-#   for more details.
```

Comment:
I think this note should be kept, to avoid again desync of git and ibs.

Reply:
right. will add it back